wrong import paths to generated data wrappers on Windows

The TS generator on Windows generates incorrect import paths to TS wrappers for data classes.

Example on Windows (incorrect)
```ts
// @ts-ignore
import client from './connect-client.default';
import Product from '..\.\com\vaadin\polyglotdx\data\Product';

export function list(): Promise<Array<Product | null> | null> {
  return client.call('ProductEndpoint', 'list', undefined, {requireCredentials: false});
}
```

Same file generated on Mac OS (correct)
```ts
// @ts-ignore
import client from './connect-client.default';
import Product from './com/vaadin/polyglotdx/data/Product';

export function list(): Promise<Array<Product | null> | null> {
  return client.call('ProductEndpoint', 'list', undefined, {requireCredentials: false});
}
```

Steps to reproduce (on a Windows 10 machine):
 - download and extract the attached project [polyglot-dx-0.9.2-1c-show-backend-data.zip](https://github.com/vaadin/vaadin-connect/files/2999222/polyglot-dx-0.9.2-1c-show-backend-data.zip)
 - in the project older run `npm install && npm start`
 - see that the build fails with the following error message:
```
ERROR in [at-loader] ./frontend/generated/ProductEndpoint.ts:3:21
    TS2307: Cannot find module '...com♂aadinpolyglotdxdataProduct'.

 ? Vaadin Connect There are compilation errors in the frontend code
```

Expected behavior:
 - the build passes OK (which is the case on on Mac OS)

--- a/vaadin-connect-maven-plugin/src/main/java/com/vaadin/connect/plugin/generator/VaadinConnectTsGenerator.java
+++ b/vaadin-connect-maven-plugin/src/main/java/com/vaadin/connect/plugin/generator/VaadinConnectTsGenerator.java
@@ -651,10 +651,15 @@ public class VaadinConnectTsGenerator extends AbstractTypeScriptClientCodegen {
       String relativizedPath = Paths
           .get(relativePathFromGeneratedFolderToCurrentFile)
           .relativize(Paths.get(importPath)).toString();
-      relativizedPath = StringUtils.prependIfMissing(relativizedPath, "./", ".",
-          "/");
+      relativizedPath = relativePathToNodeImport(relativizedPath);
       anImport.put("importPath", relativizedPath);
     }
+  }
+
+  private String relativePathToNodeImport(String relativePath) {
+    // on Windows Node imports should still use Unix-style path separator
+    relativePath = relativePath.replace("\\", "/");
+    return StringUtils.prependIfMissing(relativePath, "./", ".", "/");
   }
 
   private String getUniqueNameFromQualifiedName(Set<String> usedNames,

